rbx_binary/Cargo.toml

```diff
@@ -14,8 +14,9 @@ edition = "2018"
 byteorder = "1.2.7"
 log = "0.4.6"
 lz4 = "1.23.1"
-rbx_dom_weak = "1.10.0"
-rbx_reflection = "3.3.408"
+rbx_dom_weak = { package = "rbx_dom_weak2", path = "../rbx_dom_weak-v2" }
```

Comment:
Make sure you attach versions to these if you plan on `cargo publish`ing, otherwise it won't let you.